mula/scheduler/schedulers/schedulers/boefje.py

```diff
@@ -194,6 +194,7 @@ def push_tasks_for_scan_profile_mutations(self, body: bytes) -> None:
             thread_name_prefix=f"BoefjeScheduler-TPE-{self.scheduler_id}-mutations"
         ) as executor:
             for boefje in boefjes:
+                # Is the boefje allowed to run on the ooi?
```

Comment:
If most boefjes are not allowed to run, or otherwise fail to produce a job, maybe we can move this filtering up to line 183 where we presumably query the database or list of boefjes?

Reply:
This however checks specifically checks of based on the scan level the boefje is able to run on the supplied ooi

Reply:
I get that, but we could give the scan level to the query, right? It should limit the number of boefjes returned, since we are only dealing with a single OOI at a time here?

Reply:
Could be a seperate PR, not blocking for this obviously